### PR TITLE
add zeek log parser

### PR DESCRIPTION
**Pull request by syncpark**
* add option `--zeek` to convert zeek log
* change REproduce to wait if the input file is not exist.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Added
 
 - Support for x86_64-unknown-linux-musl.
+- Add an option for zeek log: `--zeek`. This option will filter out comments like `#...`, and convert `,` -> `;` then `\t` -> `,`.
```
Comment (msk) [inline CHANGELOG.md:13]: Is this necessary? REconverge should be able to parse Zeek's TSV properly.

Comment (syncpark) [reply CHANGELOG.md:13]: Without this conversion REconverge create only 1 cluster. 
After conversion, REconverge create 17 clusters.
I thought REconverge does not work like Hog.

Comment (msk) [reply CHANGELOG.md:13]: Then it is a bug in REconverge. REconverge should be able to read Zeek log files without change. What configuration did you use for CSV parsing in REconverge?

Comment (syncpark) [reply CHANGELOG.md:13]: Nothing special. Just `"event_type": "csv"`

```json
{
    "description": "REconverge Configuration File.",
    "period": 1,
    "detectors": [
        {
            "detector_id": 10,
            "rpc_addr": "127.0.0.1:38390",
            "input": "zeek_http",
            "event_type": "csv",
            "max_event_count": 10000,
            "model": "zeek_http",
            "review_model": "zeek_http",
            "classification_method": "multidimension",
            "format": [
                { "data_type": "datetime", "weight": 0,    "format": "%s%.f",   "alias": "ts" },
                { "data_type": "utf8",     "weight": 0,    "format": null,      "alias": "uid" },
                { "data_type": "ipaddr",   "weight": 0,    "format": null,      "alias": "id.orig_h" },
                { "data_type": "enum",     "weight": 0,    "format": null,      "alias": "id.orig_p" },
                { "data_type": "ipaddr",   "weight": 0,    "format": null,      "alias": "id.resp_h" },

                { "data_type": "enum", "weight": 0,        "format": null, "alias": "id.resp_p" },
                { "data_type": "int64","weight": 0,        "format": null, "alias": "trans_depth" },
                { "data_type": "enum", "weight": 0,        "format": null, "alias": "method"},
                { "data_type": "enum", "weight": 0,        "format": null, "alias": "host"},
                { "data_type": "enum", "weight": 2147483647, "format": null, "alias": "uri"},

                { "data_type": "enum", "weight": 0, "format": null, "alias": "referer" },
                { "data_type": "utf8", "weight": 0, "format": null, "alias": "version" },
                { "data_type": "enum", "weight": 0, "format": null, "alias": "user_agent" },
                { "data_type": "utf8", "weight": 0, "format": null, "alias": "origin" },
                { "data_type": "int64","weight": 0, "format": null, "alias": "request_body_len" },

                { "data_type": "int64","weight": 0, "format": null, "alias": "response_body_len" },
                { "data_type": "int64","weight": 0, "format": null, "alias": "status_code" },
                { "data_type": "utf8", "weight": 0, "format": null, "alias": "status_msg" },
                { "data_type": "int64","weight": 0, "format": null, "alias": "info_code" },
                { "data_type": "utf8", "weight": 0, "format": null, "alias": "info_msg" },

                { "data_type": "enum", "weight": 0, "format": null, "alias": "tags" },
                { "data_type": "utf8", "weight": 0, "format": null, "alias": "username" },
                { "data_type": "utf8", "weight": 0, "format": null, "alias": "password" },
                { "data_type": "utf8", "weight": 0, "format": null, "alias": "proxied" },
                { "data_type": "utf8", "weight": 0, "format": null, "alias": "orig_fuids" },

                { "data_type": "utf8", "weight": 0, "format": null, "alias": "orig_filenames" },
                { "data_type": "utf8", "weight": 0, "format": null, "alias": "orig_mime_types" },
                { "data_type": "utf8", "weight": 0, "format": null, "alias": "resp_fuids" },
                { "data_type": "utf8", "weight": 0, "format": null, "alias": "resp_filenames" },
                { "data_type": "utf8", "weight": 0, "format": null, "alias": "resp_mime_types" }
            ],
            "stats": false,
            "enum_dimension_default": 512,
            "time_column": 0,
            "count_columns": [
                0,
                1
            ],
            "eps": 3,
            "algorithm": "dbscan",
            "intervals": "[60]",
            "numbers": "[30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30]",
            "output": "/Users/dkpark/work/data/output",
        }
    ]
}
```

Comment (msk) [reply CHANGELOG.md:13]: Did you set the delimiter to TAB during training? See REconverge's CHANGELOG for `--csv_config`.

Comment (syncpark) [reply CHANGELOG.md:13]: I tested several times with '--csv_config' option.

* With tab separated Zeek log and `--csv_config '{ "delimiter": 9, "quote": 34 }'` option, REconverge create only 1 cluster.
```
petabi@sun:/data/work/test$ RUST_LOG=debug ./reconverge -c reconverge_config.json train -i zeek-log/http.log.training -t csv -o output -m multidimension -M httplog --csv_config '{ "delimiter": 9, "quote": 34 }'
2021-12-28T08:56:39.159300Z  INFO reconverge: Version 0.29.0-alpha.2
2021-12-28T08:56:39.159766Z  INFO reconverge::remake::cluster: Parsing input
2021-12-28T08:56:39.160071Z  INFO reconverge::remake::cluster::multidimension: Starting 32 parsers
2021-12-28T08:56:40.086403Z  INFO reconverge::remake::cluster::multidimension: Clustering
2021-12-28T08:56:44.906337Z  INFO reconverge::remake::cluster: Saving clusters
2021-12-28T08:56:45.014263Z  INFO reconverge::remake::cluster::multidimension: Processed 100000 event(s).
2021-12-28T08:56:45.014273Z  INFO reconverge::remake::cluster::multidimension: Created 1 cluster(s).
2021-12-28T08:56:45.014288Z  INFO reconverge::remake::cluster::multidimension: 0 outlier(s).
2021-12-28T08:56:45.040415Z  INFO reconverge::remake::cluster: Saving the model
```

* With comma separated Zeek log and with or without `--csv_config '{ "delimiter": 44, "quote": 34 }'`, REconverge create multiple clusters
```
petabi@sun:/data/work/test$ RUST_LOG=debug ./reconverge -c reconverge_config.json train -i zeek-log/http.log.samples -t csv -o output -m multidimension -M httplog --csv_config '{ "delimiter": 44, "quote": 34 }'
2021-12-28T08:59:31.215166Z  INFO reconverge: Version 0.29.0-alpha.2
2021-12-28T08:59:31.215640Z  INFO reconverge::remake::cluster: Parsing input
2021-12-28T08:59:31.215941Z  INFO reconverge::remake::cluster::multidimension: Starting 32 parsers
2021-12-28T08:59:33.194315Z  INFO reconverge::remake::cluster::multidimension: Clustering
2021-12-28T09:00:05.359406Z  INFO reconverge::remake::cluster: Saving clusters
2021-12-28T09:00:05.467703Z  INFO reconverge::remake::cluster::multidimension: Processed 99524 event(s).
2021-12-28T09:00:05.467712Z  INFO reconverge::remake::cluster::multidimension: Created 34 cluster(s).
2021-12-28T09:00:05.467715Z  INFO reconverge::remake::cluster::multidimension: 0 outlier(s).
2021-12-28T09:00:05.488468Z  INFO reconverge::remake::cluster: Saving the model
```

**Pull request closed without merging**